### ISSUE TITLE
Doesn't build on recent Linux v5.10 revisions because of refactored zstd

### ISSUE THREAD

**Issue by tsjk** [edward6/reiser4#8]
```include/linux/zstd.h``` has changed significantly, and several ```ZSTD_*``` references from Reiser4 now results in compilation errors.

**Comment by edward6** [comment]
Which exactly revision of 5.10 causes the problem? With 5.10.17 for me it is built just fine..

Thanks,
Eduard.

**Comment by tsjk** [comment]
That is strange. I tried with v5.10.15 (or 5.10-pf12 from https://gitlab.com/post-factum/pf-kernel/, with Gentoo's genpatches).
But, perhaps I did a mistake here. I integrated a modified version of ZFS in my kernel (https://github.com/BrainSlayer/zfs; because the mainstream one cannot be built in at the moment), and perhaps their upgraded ZSTD did something to confuse other things as well. I'm sorry for having wasted time.
